tests/integration/test_plugins.py

```diff
@@ -88,18 +90,17 @@
 TIMESCALEDB_EXTENSION_STATEMENT = "CREATE TABLE test_timescaledb (time TIMESTAMPTZ NOT NULL); SELECT create_hypertable('test_timescaledb', 'time');"
 
 
-@pytest.mark.group(1)
 @pytest.mark.abort_on_fail
-async def test_plugins(ops_test: OpsTest) -> None:
+async def test_plugins(ops_test: OpsTest, charm) -> None:
     """Build and deploy one unit of PostgreSQL and then test the available plugins."""
     # Build and deploy the PostgreSQL charm.
     async with ops_test.fast_forward():
-        charm = await ops_test.build_charm(".")
         await ops_test.model.deploy(
             charm,
             num_units=2,
             base=CHARM_BASE,
-            config={"profile": "testing"},
+            # TODO Figure out how to deal with pgaudit
+            config={"profile": "testing", "plugin_audit_enable": "False"},
```

Comment:
Generally disable pgaudit for this test, until we decide how to handle incompatibilities.

Reply:
Talk to upstream to address incompatibility ASAP?

Reply:
Upstream bug is https://github.com/timescale/timescaledb/issues/7667. There should be fixes for this in Timescale 2.18.2, but there seems to be additional issues on pgaudit's side. We can update our PPA to 2.18.2 and see if it helps.